website/src/content/blog/2025-05-02-typespec-1-0-GA/typespec_1_0_GA.md

```diff
@@ -0,0 +1,161 @@
+---
+slug: typespec-1-0-GA-release
+title: "TypeSpec 1.0 GA: API First, Made Practical"
+image: ./TypeSpec_1_0_GA.png
+description: "TypeSpec 1.0 is now generally available! This Microsoft-built, community-supported open-source project provides a powerful API modeling language that streamlines API-first development, enabling the generation of server code, client libraries, schemas, and documentation from a single source of truth."
+publishDate: 2025-05-02
+authors:
+  - name: Mario Guerra
+    title: Senior Product Manager @ Microsoft
+    avatar: assets/img/authors/mario_guerra.png
+tags:
+  - release
+  - announcement
+---
+
+Today, we're thrilled to announce the general availability of TypeSpec 1.0 GA! This milestone marks the transition of TypeSpec's core components from release candidate to production-ready status. TypeSpec is a Microsoft-built, community-supported project that makes API-first development truly practical by empowering you to define your APIs in a concise, human-readable format and generate all the artifacts you need—from OpenAPI specifications to client libraries and server code scaffolding—from a single source of truth.
+
+The Azure SDK team has considered API-first development a best practice for a long time, but implementing it effectively has remained challenging. With TypeSpec 1.0 GA, we're delivering a production-ready solution that bridges the gap between API design principles and real-world implementation, making API-first development practical for teams of all sizes.
```

Comment:
While we call it internally GA, I think the community norm is "TypeSpec 1.0 stable"

Reply:
I'll just refer to it as "TypeSpec 1.0" and drop the qualifiers altogether.